### ISSUE TITLE
Creature not loading into extractor v0.26.3

### ISSUE THREAD

**Issue by geneticbits** [cadon/ARKStatsExtractor#651]
I created a values file for some modded dinos on my server. It used to load  into the extractor/stat tester fine. But since I updated to v0.26.3 I can't load those dinos in the extractor/tester screens anymore. They are in the list, but after I select them, the actual screen where you put in the stats doesn't come back like it does with vanilla dinos. 
![asb](https://user-images.githubusercontent.com/25036257/37877456-a58de85a-3029-11e8-9066-08f9d53ae9c7.PNG)

**Comment by cadon** [comment]
Can you click on the entry in the list on the left?

**Comment by geneticbits** [comment]
No. I've tried left clicking. Double clicking. Right clicking. None have worked. 


 
 
  On Sun, Mar 25, 2018 at 6:52 PM, cadon<notifications@github.com> wrote:   
Can you click on the entry in the list on the left?

**Comment by cadon** [comment]
Does it work if you press enter while in the text-input? Can you click on the pygmyGriphon-entry in the box on the bottom-right?

**Comment by geneticbits** [comment]
Nope. Just tried pressing enter in the text box, while having the pygmy selected in the left pane, and with pygmy selected in the "in library" pane. I tried left, double and right clicking all three places too. :/ 
 
  On Sun, Mar 25, 2018 at 6:56 PM, cadon<notifications@github.com> wrote:   
Does it work if you press enter while in the text-input? Can you click on the pygmyGriphon-entry in the box on the bottom-right?

**Comment by cadon** [comment]
Can you upload the gryphons.json-file so I can test it further?

**Comment by geneticbits** [comment]
On Sunday, March 25, 2018, 7:04:51 PM EDT, cadon <notifications@github.com> wrote:  
 
 
Can you upload the gryphons.json-file so I can test it further?

**Comment by geneticbits** [comment]
[gryphons.zip](https://github.com/cadon/ARKStatsExtractor/files/1845922/gryphons.zip)
 It wouldn't let my attach the json, so I zipped it. I can also send it as a .txt apparently. Or is there a better way to send it?

**Comment by cadon** [comment]
Thanks. Found and fixed the issue. It was related to the camelCase of these creatures. The new release will be out soon.
